### PR TITLE
unify S/4HANA, HANA and NetWeaver spelling in documentation

### PR DESCRIPTION
**Pull request by melzer-b1**
This is an update for the documentation to unify the spelling of the S/4HANA and NetWeaver products.

### CODE REVIEW

**Comment by yeoldegrove** [comment]
@angelabriel What do you think about mentioning "Netweaver or S/4HANA" every time on every occasion?
It could get a bit overwhelming I guess...
I would be fine with @melzer-b1's approach for now.

File: doc/ip_autogeneration.md [hunk 1]
```diff
@@ -30,13 +31,13 @@ Example based on `10.74.0.0/16` vnet address range and `10.74.0.0/24` subnet add
...
+| HANA cluster vip | `hana_cluster_vip` | `10.74.0.12` | Only used if HA is enabled in HANA |
+| HANA cluster vip secondary | `hana_cluster_vip_secondary` | `10.74.0.13` | Only used if the Active/Active setup is used |
 | DRBD ips | `drbd_ips` | `10.74.0.20`, `10.74.0.21` ||
 | DRBD cluster vip | `drbd_cluster_vip` | `10.74.0.22` ||
-| Netweaver ips | `netweaver_ips` | `10.74.0.30`, `10.74.0.31`, `10.74.0.32`, `10.74.0.33` | Addresses for the ASCS, ERS, PAS and AAS. The sequence will continue if there are more AAS machines |
-| Netweaver virtual ips | `netweaver_virtual_ips` | `10.74.0.34`, `10.74.0.35`, `10.74.0.36`, `192.168.135.37` | The 1st virtual address will be the next in the sequence of the regular Netweaver addresses |
+| S/4HANA or NetWeaver ips | `netweaver_ips` | `10.74.0.30`, `10.74.0.31`, `10.74.0.32`, `10.74.0.33` | Addresses for the ASCS, ERS, PAS and AAS. The sequence will continue if there are more AAS machines |
+| S/4HANA or NetWeaver virtual ips | `netweaver_virtual_ips` | `10.74.0.34`, `10.74.0.35`, `10.74.0.36`, `192.168.135.37` | The 1st virtual address will be the next in the sequence of the regular S/4HANA NetWeaver addresses |
```
Comment (angelabriel) [inline doc/ip_autogeneration.md:40]: missing the 'or' between S/4HANA and NetWeaver

**Review by angelabriel — CHANGES_REQUESTED** [review]
Looks good, but there is one missing 'or'. Please be so kind and add this too.

File: doc/ip_autogeneration.md (hunk 1 above)
Comment (melzer-b1) [reply doc/ip_autogeneration.md]: It is now corrected. Thank you.

**Review by angelabriel — APPROVED** [review]
Thanks for the 'or' :-)
Looks good to me. Approved.

**Comment by angelabriel** [comment]
> @angelabriel What do you think about mentioning "Netweaver or S/4HANA" every time on every occasion?
> It could get a bit overwhelming I guess...
> I would be fine with @melzer-b1's approach for now.
@yeoldegrove yes, it's fine for me too. I just approved.

**Pull request merged**